README.md

```diff
@@ -19,6 +19,27 @@ structures.
 
 Check out our [short-term roadmap](#short-term-roadmap) below for where we're heading!
 
+## Documentation
+
+- Setup
```

Comment:
I don't really want to keep this updated manually as opposed to just linking people to docs.tigerbeetle.com?